src/__tests__/__fixtures__/typeScriptIntegrationProject/src/steps/fetchAccounts.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  createIntegrationEntity,
+  IntegrationStepExecutionContext,
+} from '../../../../../framework';
+
+export default {
+  id: 'fetch-accounts',
+  name: 'Fetch Accounts',
+  types: ['my_account'],
+  // executionHandler: () => {},
```

Comment:
This looks like an old comment.

Reply:
I didn't remove that when I was testing something, good catch.